spec/dummy/db/migrate/20160125083537_create_models.rb

```diff
@@ -1,22 +0,0 @@
-class CreateModels < ActiveRecord::Migration
-  def change
-    create_table :comments do |t|
-      t.string :article_id
-      t.belongs_to :author
-      t.belongs_to :reviewing_user, references: :user
-    end
-
-    create_table :users do |t|
-    end
-
-    create_table :articles do |t|
-      t.string :external_id, null: false
-      t.references :author
-      t.string :blank_value
-    end
-
-    create_table :tags do |t|
-      t.references :taggable, polymorphic: true
-    end
-  end
-end
```

Comment:
Did not understand this change. We have the models, but are not keeping the DDL commands for the same?

Reply:
@soni-sahu Because this is for the test database, migrations are not required -- test databases can and should be recreated from scratch.

Reply:
Got it, thank you!